### PR TITLE
DOC: Fix reference, old file was separated last year

### PR DESCRIPTION
**Pull request by phofl**
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

The comment referenced a file deleted last year after separating the tests into a few files. I referenced the correct folder again

### CODE REVIEW

File: pandas/tests/io/formats/test_to_excel.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 """Tests formatting as writer-agnostic ExcelCells
 
-ExcelFormatter is tested implicitly in pandas/tests/io/test_excel.py
+ExcelFormatter is tested implicitly in pandas/tests/io/excel tests
```
Comment (dsaxton) [inline pandas/tests/io/formats/test_to_excel.py:3]: Could also change this to the actual file pandas/tests/io/excel/test_style.py

Comment (phofl) [reply pandas/tests/io/formats/test_to_excel.py:3]: Had the same idea initially, but we are running through this class from all files except test_odf and test_readers. I though referencing the folder would be better in this case.

Should I change it?

Comment (dsaxton) [reply pandas/tests/io/formats/test_to_excel.py:3]: Up to you, either works (this also wouldn't have to change again if / when tests get moved around)

Comment (phofl) [reply pandas/tests/io/formats/test_to_excel.py:3]: Yes, had similar thoughts. I would keep it like it is now then. Thx

Comment (dsaxton) [reply pandas/tests/io/formats/test_to_excel.py:3]: Very nit-picky, but you might even drop the last word "tests" as redundant

**Review by dsaxton — APPROVED** [review]
Small nit-pick, otherwise good find / clean-up

**Comment by phofl** [comment]
> 
> 
> Small nit-pick, otherwise good find / clean-up

Done :)

Thanks

**Review by MarcoGorelli — APPROVED** [review]

**Comment by jreback** [comment]
thanks @phofl

**Pull request merged**